libdistopia/libdistopia/lib/vector_triple.h

```diff
@@ -140,6 +140,17 @@ class VectorTriple
         DeinterleaveIdx(v_arr, x, y, z);
     }
 
+    /** \brief Loads a single coordinate and broadcasts across the vectors
+     *  i.e. all values in the x array are source[0], all y are source[1], all z are source[2]
+     *
+     * @param source start of coordinates to load
+     */
+    void load_single(const ScalarT *source) {
```

Comment:
@hmacdope nearly called this `set`, or is `load_single` enough?

Reply:
Load single is fine by me.